support.md

```diff
@@ -1,11 +1,15 @@
-# Support
+# Adobe Developer Console Support
 
-Learn where to ask questions, report bugs, make feature requests, and spark discussions.
+If you have questions about how to use Adobe Developer Console or you are looking to report bugs, make feature requests, and more, Console provides multiple resources that can help.
 
-## [FAQ](faq.md)
+## Frequently Asked Questions
 
-A knowledge base of frequently asked questions. This is a great starting place to find an answer before asking your question.
+Access a knowledge base of frequently asked questions and find answers to some of the most common questions asked by Adobe Developer Console users. This is a great place to start when needing answer to questions. 
 
-## [Adobe Forums](https://forums.adobe.com/community/adobe-io/adobe-io-console)
+Visit the [Adobe Developer Console FAQ](faq.md) to learn more.
 
-This is the place to report bugs, ask questions, make feature requests, or start a discussion. You'll find official Adobe developers and community members available to help you out.
+## Adobe Forums
+
+To report bugs, ask questions, make feature requests, or start a discussion, visit the [Adobe Forums](https://forums.adobe.com/community/adobe-io/adobe-io-console). 
+
```

Comment:
looks like the Form link redirects to https://experienceleaguecommunities.adobe.com/t5/adobe-i-o-console/ct-p/adobe-io-console
Can we just use that link? Or maybe create a goURL?

Reply:
Appears that they shifted the forums into Experience League, so I made a new goURL - `devs_console_exl` - and then we can adjust that later once ExL changes their URL as it currently says "io-console". I also updated this whole section to talk about Experience League and community instead of Adobe Forums.